lib/Indexer/Extension/Command/IndexCleanCommand.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace Phpactor\Indexer\Extension\Command;
+
+use Phpactor\Indexer\Model\Index\IndexInfo;
+use Phpactor\Indexer\Util\Filesystem as PhpactorFilesystem;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class IndexCleanCommand extends Command
+{
+    public const CLEAN_ALL = 'clean-all';
+    private const INDEX_TO_CLEAN = 'index-to-clean';
+
+    private string $indexDirectory;
+
+    private Filesystem $filesystem;
+
+    public function __construct(string $indexDirectory, Filesystem $filesystem)
+    {
+        parent::__construct();
+        $this->indexDirectory = $indexDirectory;
+        $this->filesystem = $filesystem;
+    }
+
+    protected function configure(): void
+    {
+        $this->setDescription('Removing a project index from the cache');
+        $this->setHelp(sprintf(<<<DOCS
+            === Non interactive version ===
+            Listing the available indices
+                bin/console index:clean --no-interaction
+
+            Removing an index by name
+                bin/console index:clean <index name>
+
+            Removing an index by the number in the list view:
+                bin/console index:clean <index number>
+
+            Removing all indicies
+                bin/console index:clean %s
```

Comment:
%s?

Reply:
Yeah this is an `sprintf` Argument, and it will be filled with the `CLEAN_ALL` option.